src/components/Header/Header.js

```diff
@@ -22,8 +19,8 @@ class Header extends Component {
     super(props);
 
     this.toggleNavbar = this.toggleNavbar.bind(this);
+    this.modalOpen = this.modalOpen.bind(this);
     this.modalToggle = this.modalToggle.bind(this);
```

Comment:
this binding should be removed since you changed this function name to modalOpen